.github/workflows/beekeeper.yml

```diff
@@ -56,89 +56,17 @@ jobs:
       - name: Set local cluster
         run: |
           make beelocal ACTION=add-hosts
-          timeout 10m make deploylocal BEEKEEPER_CLUSTER=local-dns
+          timeout 10m make deploylocal BEEKEEPER_CLUSTER=local-dns BEEKEEPER_BRANCH=auth-health-check
       - name: Test pingpong
         id: pingpong-1
         run: until beekeeper check --cluster-name local-dns --checks ci-pingpong; do echo "waiting for pingpong..."; sleep .3; done
-      - name: Test fullconnectivity
-        id: fullconnectivity-1
-        run: beekeeper check --cluster-name local-dns --checks=ci-full-connectivity
-      - name: Test settlements
-        id: settlements-1
-        run: beekeeper check --cluster-name local-dns --checks=ci-settlements
-      - name: Sleep for time allowance to replenish
-        run: sleep 2
-      - name: Test pss
-        id: pss
-        run: beekeeper check --cluster-name local-dns --checks=ci-pss
-      - name: Test soc
-        id: soc
-        run: beekeeper check --cluster-name local-dns --checks=ci-soc
-      - name: Test pushsync (chunks)
-        id: pushsync-chunks-1
-        run: beekeeper check --cluster-name local-dns --checks=ci-pushsync-chunks
-      - name: Test pushsync (light mode chunks)
-        id: pushsync-chunks-2
-        run: beekeeper check --cluster-name local-dns --checks=ci-pushsync-light-chunks
-      - name: Test retrieval
-        id: retrieval-1
-        run: beekeeper check --cluster-name local-dns --checks=ci-retrieval
-      - name: Test manifest
-        id: manifest-1
-        run: beekeeper check --cluster-name local-dns --checks=ci-manifest
-      - name: Test content availability
-        id: content-availability
-        run: beekeeper check --cluster-name local-dns --checks=ci-content-availability
+      - name: Test authenticated health
```

Comment:
Should we wait for new beekeeper version to be released with the ci-auth check and to get back ?

Reply:
yes, I will mark it as draft and merge it only after BK changes are in.